cluster/charts/rook-ceph/values.yaml

```diff
@@ -53,10 +53,23 @@ rbacEnable: true
 ##
 pspEnable: true
 
-## Settings for whether to disable the drivers or other daemons if they are not needed
-enableCsiRbdDriver: true
-enableCsiCephfsDriver: true
-enableCsiGrpcMetrics: true
+## Settings for whether to disable the drivers or other daemons if they are not
+## needed
+csi:
+  enableRbdDriver: true
+  enableCephfsDriver: true
+  enableGrpcMetrics: true
+  #cephcsi:
```

Comment:
Do we anticipate other settings next to the images? It might be better to keep it simple and just have the flat list of settings under `csi`. Or if we keep it like this, we need more checks above as mentioned in the separate comment.

Reply:
yes, which this required to updating the options specific to the sidecar container and csi container.